features/polyExplorer/src/static/images/stories/messenger/card-image.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="328" height="169" viewBox="0 0 328 169" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg viewBox="0 0 328 169" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Can't you also set the viewBox in the CSS file so that it's not repeated twice? Ditto for the `fill` value.

Reply:
I don't know! Is there a CSS-property as viewbox?

Reply:
(googles pretty fast...)

Reply:
... no https://stackoverflow.com/a/46108725/891440
You can set it via JS properties, though.

Reply:
Also, this: https://stackoverflow.com/a/20720935/891440 you can have CSS files for SVGs, possibly external to them. Not sure we would want to do that.